CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Fixed 
+- Mid rolls errorneously reported as post rolls when midroll's schedule time > the duration of the adbreak 
+- Adhelper testcases to use the source duration instead
```

Comment:
No need for adding notes about test cases

Reply:
Updated https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/commit/19815ff3a9dd1b44258aa68ef0c0c194044e6738